_includes/_subscribe-buttons.html

```diff
@@ -60,7 +60,7 @@ <h2 class="component-header flush-top">Subscribe</h2>
         {% endif %}
         {% if include.content_type.stitcher_url.size > 1 %}
         <li>
-          <a href="{{ include.content_type.stitcher_url }}" role="button" class="btn btn-cyan btn-subscribe soft-quarter-ends d-flex justify-content-between">
+          <a href="{{ include.content_type.stitcher_url }}" role="button" class="btn btn-cyan btn-subscribe soft-quarter-ends d-flex justify-content-between" onclick="({'event': 'media-subscribe-button-pushed', 'event_action': 'subscribed'})">
```

Comment:
This isn't actually pushing to the `dataLayer`. Should we abstract this logic into a JS file so we don't have 4 instances of the same event handler?